modules/ROOT/pages/java-se.adoc

```diff
@@ -10,18 +10,65 @@
-For more information, see https://openliberty.io/blog/2019/02/06/java-11.html[Open Liberty and Java 11].
+There are two commonly used Java Virtual Machine (JVM) implementations: Eclipse OpenJ9 and HotSpot.
+They provide similar throughput performance, but make different tradeoffs regarding memory footprint and startup.
+As a result Open Liberty recommends using a Java distribution that packages Eclipse OpenJ9 because it has a faster start time and uses less memory.
+The main Java distribution using Eclipse OpenJ9 is IBM Semeru.
+IBM Semeru provides uncertified Open Edition releases under a GPLv2 with classpath exception license, and Certified Edition releases under an IBM license.
 
-Due to differences between Java SE 8 and Java SE 11, an Open Liberty application that runs on Java SE 8 might not run on Java SE 11. For more information, see https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-C25E2B1D-6C24-4403-8540-CFEA875B994A[Oracle Java SE 11 migration guide].
+== Supported Java SE releases
 
-== Java SE 15
-Open Liberty runs on any recent Java SE 15 release. Java SE 15 is not a long-term supported release and standard support is scheduled to end in March 2021. Keep in mind, if you download your Java SDK from https://adoptopenjdk.net/index.html?variant=openjdk15&jvmVariant=openj9[AdoptOpenJDK], https://www.eclipse.org/openj9/[Eclipse OpenJ9] has a better memory footprint and startup profile than https://openjdk.java.net/groups/hotspot/[HotSpot]. For more information, see Open Liberty and Java 11.
+.Supported Java Releases
+[%header,cols=5]
+|===
+|Java version
+|LTS Release
+|Eclipse OpenJ9 Download
+|HotSpot Download
+|Migration Advice
+
+|8
+|&check;
+|https://developer.ibm.com/languages/java/semeru-runtimes/downloads/?version=8[IBM Semeru 8]
+|https://adoptium.net/?variant=openjdk8&jvmVariant=hotspot[Eclipse Adoptium 8]
+|
+
+|11
+|&check;
+|https://developer.ibm.com/languages/java/semeru-runtimes/downloads/?version=11[IBM Semeru 11]
+|https://adoptium.net/?variant=openjdk11&jvmVariant=hotspot[Eclipse Adoptium 11]
+|https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-C25E2B1D-6C24-4403-8540-CFEA875B994A[Java SE 11 migration guide]
+
+|17
+|&check;
+|
+|https://adoptium.net/?variant=openjdk17&jvmVariant=hotspot[Eclipse Adoptium 17]
+|https://docs.oracle.com/en/java/javase/17/migrate/toc.htm[Java SE 17 migration guide]
+|===
+
+== Migration tools
+
+In addition to the Java migration guides there are also some tools available that can provide advice targeted to your applications.
+The https://www.ibm.com/support/pages/node/6250913[Migration Toolkit for Application Binaries] can scan a WAR or EAR file and provide advice on migrating to new Java SE versions.
```

Comment:
The tool will also scan JAR files but you must use the `--analyze` parameter below.  Suggestion:

The https://www.ibm.com/support/pages/node/6250913[Migration Toolkit for Application Binaries] can scan WAR,  EAR, or JAR files and provide advice on migrating to new Java SE versions.